odoo/addons/base/models/res_users_view.py

```diff
@@ -1,14 +1,43 @@
+from odoo import api, fields, models, tools, _
 
-from odoo import api, fields, models, tools
+
+class ResGroupsSection(models.Model):
+    _name = 'res.groups.section'
+    _description = "Sections"
+    _order = 'sequence,name'
+
+    name = fields.Char(string='Name', required=True, translate=True)
+    sequence = fields.Integer(string='Sequence', default=100)
+    privilege_ids = fields.One2many('res.groups.privilege', 'section_id', string='Categories')
+
+
+class ResGroupsPrivilege(models.Model):
+    _name = 'res.groups.privilege'
+    _description = "Privileges"
+    _order = 'sequence,name'
+
+    name = fields.Char(string='Name', required=True, translate=True)
+    description = fields.Text(string='Description', compute="_compute_description")
+    placeholder = fields.Char(string='Placeholder', default="No", help="Text that is displayed as placeholder in the selection field of the user form.")
+    sequence = fields.Integer(string='Sequence', default=100)
+    section_id = fields.Many2one('res.groups.section', string='Group Section')
+    group_ids = fields.One2many('res.groups', 'privilege_id', string='Groups')
+
+    @api.depends('group_ids')
+    def _compute_description(self):
+        for privilege in self:
+            privilege.description = '\n'.join(f'{g.name}: {g.comment}' for g in privilege.group_ids if g.comment)
```

Comment:
That's weird to ignore `group_ids` that din't have comments. We can still show their names :thinking:

Reply:
If you want to be 100 % correct, you should use self.env._ here since `g.comment` is translated and the pattern '%s: %s' may change depending of the lang.

Reply:
This is the help in the view, users have the selector with the values. The need is to see the eventual details of the groups.